commands_to_load/Administration.py

```diff
@@ -11,18 +11,29 @@ class Administration():
 	def __init__(self, bot):
 		self.bot = bot
 
+	def realCog(self, cog):
+		for name in cogs:
+			if name["name"] == cog:
+				return [True, name["folder"]]
+		return [False]
+
 	@commands.command()
 	async def load(self, ctx, name):
 		logger.info("[Administration load()] load command detected from "+str(ctx.message.author))
 		if ctx.message.author in discord.utils.get(ctx.guild.roles, name="Bot_manager").members:
+			folder = self.realCog(name)

```

Comment:
Python has a way to return 2 variables from a function, that will be useful here so you dont have to refer to `folder[1]` when you could just do `folder`

Reply:
same thing applies for below functions

Reply:
done